www/include/monitoring/downtime/AddDowntime.php

```diff
@@ -283,8 +283,30 @@
         )
     );
 
+    $durationScale = array(
+        'd' => 86400,
+        'h' => 3600,
+        'm' => 60,
+        's' => 1,
+    );
     $defaultDuration = 7200;
+    $defaultScale = 's';
+    if (isset($centreon->optGen['monitoring_dwt_duration']) &&
```

Comment:
```suggestion
    if (isset($centreon->optGen['monitoring_dwt_duration'])
```

Reply:
`&&` is at the end of the lines everywhere in this file, not at the beginning of the second line 😕